src/infrastructure/routes/conseillers.controller.ts

```diff
@@ -58,17 +61,46 @@ import { CreateRendezVousPayload } from './validation/rendez-vous.inputs'
 export class ConseillersController {
   constructor(
     private readonly getDetailConseillerQueryHandler: GetDetailConseillerQueryHandler,
+    private readonly getConseillerByEmailQueryHandler: GetConseillerByEmailQueryHandler,
     private readonly getJeunesByConseillerQueryHandler: GetJeunesByConseillerQueryHandler,
     private readonly getResumeActionsDesJeunesDuConseillerQueryHandler: GetResumeActionsDesJeunesDuConseillerQueryHandler,
     private readonly createActionCommandHandler: CreateActionCommandHandler,
     private readonly creerJeunePoleEmploiCommandHandler: CreerJeunePoleEmploiCommandHandler,
     private readonly sendNotificationNouveauMessage: SendNotificationNouveauMessageCommandHandler,
     private readonly getAllRendezVousConseillerQueryHandler: GetAllRendezVousConseillerQueryHandler,
-    private createRendezVousCommandHandler: CreateRendezVousCommandHandler,
-    private getDossierMiloJeuneQueryHandler: GetDossierMiloJeuneQueryHandler,
-    private creerJeuneMiloCommandHandler: CreerJeuneMiloCommandHandler
+    private readonly createRendezVousCommandHandler: CreateRendezVousCommandHandler,
```

Comment:
rappelle moi le readonly c'etait pourquoi ? :(

Reply:
Pour pas faire "accidentellement" `this.createRendezVousCommandHandler = autreChose`.